src/qt/guiutil.cpp

```diff
@@ -246,8 +246,8 @@ void openDebugLogfile()
 #endif
 }
 
-ToolTipToRichTextFilter::ToolTipToRichTextFilter(int size_threshold, QObject *parent):
-    size_threshold(size_threshold), QObject(parent)
+ToolTipToRichTextFilter::ToolTipToRichTextFilter(int size_threshold, QObject *parent) :
```

Comment:
This is duplicated in #354 but because it shouldn't have been in this commit anyway I'm going to merge it and leave this one to be rebased.